### PR TITLE
Make fields of ImportedData public

### PR DESCRIPTION
**Pull request by sparkprime**
@sbarzowski @hausdorff @eicnix

### CODE REVIEW

File: imports.go [hunk 1]
```diff
@@ -125,22 +125,22 @@ func tryPath(dir, importedPath string) (found bool, content []byte, foundHere st
 func (importer *FileImporter) Import(dir, importedPath string) *ImportedData {
 	found, content, foundHere, err := tryPath(dir, importedPath)
 	if err != nil {
-		return &ImportedData{err: err}
+		return &ImportedData{Err: err}
```
Comment (seh) [inline imports.go]: - [ ] Did you consider instead making the function signature return `(*ImportedData, error)` and removing the `Err` field from `ImportedData`?  
  That is, `Import` either returns imported data or an error. When returning an error, the `*ImportedData` value would be nil.

File: imports.go [hunk 2]
```diff
@@ -149,7 +149,7 @@ type MemoryImporter struct {
 
 func (importer *MemoryImporter) Import(dir, importedPath string) *ImportedData {
```
Comment (seh) [inline imports.go]: - [ ] Likewise here, what about returning `(*ImportedData, error)` instead?

File: imports.go (hunk 1 above)
Comment (sbarzowski) [reply imports.go]: IIRC I originally put `err` inside, because I wanted errors to be cached. But I agree this feels a bit unnatural.

Comment (sparkprime) [reply imports.go]: Have done

Comment (seh) [reply imports.go]: Nice. 👍

**Pull request merged**